Logging Action

seletz opened the pull request:
Added very simplistic $log.{info,debug,error} actions.

seletz committed:
Logging Action

Added very simplistic $log.{info,debug,error} actions.

gliechtenstein commented:
Looks great! 

One thing, I've been using `text` as an attribute name when we need to pass a textual argument. Here's the most similar example [$util.toast](https://github.com/Jasonette/JASONETTE-iOS/blob/develop/app/Jasonette/JasonUtilAction.m#L38)

For consistency's sake would you mind just renaming `message` to `text` and re-commit? I think that would be easier to remember..

seletz committed:
Log Action

Renamed input option from "message" to "text" for consistency with
other actions.

seletz committed:
Log Action

Remove unneeded protocol.

gliechtenstein commented:
There's an additional commit to add these files to the project https://github.com/Jasonette/JASONETTE-iOS/commit/2d128e1e5eacfcf36644be0b0d31a5ab686c6b4b
